roles/prometheus/templates/targets/pushgateway_exporter_target.json.j2

```diff
@@ -0,0 +1,27 @@
+[
```

Comment:
I think we should have `honour_labels: true` [per this](https://github.com/prometheus/pushgateway#about-the-job-and-instance-labels)?